qiita_db/study.py

```diff
@@ -206,6 +207,26 @@ def get_by_status(cls, status):
 
         return studies
 
+    @staticmethod
+    def get_by_portal(portal):
```

Comment:
I'm nervous about adding this kind of functions. This will make the portal types explicit instead of implicit, which is not what we want with portal types.

The way that I'm thinking this is when we invoke an iterator over all the studies, we get the ones only for the portal that is accessing. As this is getting right now, is going to require a lot of changes in the code base and make the entire system aware of the portal.

I'm strongly against this kind of functions...